fxdc/config.py

```
@@ -1,33 +1,50 @@
+    "bool": bool,
+    "list": list,
+    "dict": dict,
+}
 
 class _customclass:
     def __init__(
         self,
         classname: str,
         class_: type[TB],
+        meta_data: dict[str, dict[str, Any]],
         from_data: Optional[Callable[..., TB]] = None,
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)